cuda/matrix/coo_kernels.cu

```diff
@@ -100,12 +100,14 @@ __device__ void spmv_kernel(const size_type nnz, const size_type num_lines,
                             const IndexType *__restrict__ col,
                             const IndexType *__restrict__ row,
                             const ValueType *__restrict__ b,
-                            ValueType *__restrict__ c, Clousure scale)
+                            const size_type b_stride, ValueType *__restrict__ c,
+                            const size_type c_stride, Clousure scale)
```

Comment:
I had not seen that previously. It's minor, but technically the name should be `Closure`.